database/migrations/2024_01_03_145604_create_enrollments_table.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateEnrollmentsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('enrollments', function (Blueprint $table) {
+            $table->primary('id');
```

Comment:
This can just be `$table->id();` which should fix the issue we are seeing in the build.